src/current/v23.1/monitoring-and-alerting.md

```diff
@@ -189,6 +189,685 @@ If you rely on external tools for storing and visualizing your cluster's time-se
+          "gcPolicy": {
+            "ttlSeconds": 14400,
+            "protectionPolicies": [
+            ],
+            "ignoreStrictEnforcement": false
+          },
+          "globalReads": false,
+          "numReplicas": 9,
+          "numVoters": 0,
+          "constraints": [
+          ],
+          "voterConstraints": [
+          ],
+          "leasePreferences": [
+          ],
+          "rangefeedEnabled": false,
+          "excludeDataFromBackup": false
+        }
+      },
+      ...
+    ],
+    "overReplicated": [
+    ],
+    "violatingConstraints": [
+      {
+        "rangeDescriptor": {
+          "rangeId": "238",
+          "startKey": "9IkSYW1zdGVyZGFtAAE=",
+          "endKey": "9IkSYW1zdGVyZGFtAAESxR64UeuFQAD/gAD/AP8A/wD/AP8BgQAB",
+          "internalReplicas": [
+            {
+              "nodeId": 9,
+              "storeId": 9,
+              "replicaId": 5,
+              "type": 0
+            },
+            {
+              "nodeId": 7,
+              "storeId": 7,
+              "replicaId": 4,
+              "type": 0
+            },
+            {
+              "nodeId": 8,
+              "storeId": 8,
+              "replicaId": 3,
+              "type": 0
+            }
+          ],
+          "nextReplicaId": 6,
+          "generation": "48",
+          "stickyBit": {
+            "wallTime": "0",
+            "logical": 0,
+            "synthetic": false
+          }
+        },
+        "config": {
+          "rangeMinBytes": "134217728",
+          "rangeMaxBytes": "536870912",
+          "gcPolicy": {
+            "ttlSeconds": 14400,
+            "protectionPolicies": [
+            ],
+            "ignoreStrictEnforcement": false
+          },
+          "globalReads": false,
+          "numReplicas": 9,
+          "numVoters": 0,
+          "constraints": [
+            {
+              "numReplicas": 0,
+              "constraints": [
+                {
+                  "type": 0,
+                  "key": "region",
+                  "value": "europe-west1"
+                }
+              ]
+            }
+          ],
+          "voterConstraints": [
+          ],
+          "leasePreferences": [
+          ],
+          "rangefeedEnabled": false,
+          "excludeDataFromBackup": false
+        }
+      },
+      ...
+    ],
+    "unavailable": [
+    ],
+    "unavailableNodeIds": [
+    ]
+  }
+}
+~~~
+
+##### Replication status - ranges in critical localities
+
+The following example assumes you are running a fresh, local [`cockroach demo`](cockroach-demo.html) cluster started using the following command:
+
+{% include_cached copy-clipboard.html %}
+~~~ shell
+cockroach demo --geo-partitioned-replicas --insecure
+~~~
+
+By default, this geo-distributed demo cluster will not have any critical [nodes](architecture/overview.html#node). A node is 'critical' if removing that node were to result in [replicas](architecture/overview.html#architecture-replica) becoming unavailable.
```

Comment:
```suggestion
By default, this geo-distributed demo cluster will not have any critical [nodes](architecture/overview.html#node). A node is _critical_ if removing that node would result in [replicas](architecture/overview.html#architecture-replica) becoming unavailable.
```

Reply:
This slightly differs from the earlier description, which is about nodes becoming unreachable. That seems to cover more scenarios. If it's accurate, I'd suggest using the same verbiage here.

Reply:
k, updated to use the same language here